Gemfile

```diff
@@ -25,6 +25,7 @@ gem "manageiq-gems-pending", ">0", :require => 'manageiq-gems-pending', :git =>
 # Modified gems for gems-pending.  Setting sources here since they are git references
 gem "handsoap", "~>0.2.5", :require => false, :git => "https://github.com/ManageIQ/handsoap.git", :tag => "v0.2.5-5"
 gem "rubywbem",            :require => false, :git => "https://github.com/ManageIQ/rubywbem.git", :branch => "rubywbem_0_1_0"
+gem "fog-google",          :require => false, :git => "https://github.com/juliancheal/fog-google", :branch => "proxy_support_part_deux"
```

Comment:
This is not modified for gems pending (as per the comment).  Please put this in down on line 110.